npm build fails

I've just cloned the repo and tried to build it with Docker. It failed with the following log: [build-error.log](https://github.com/timalanfarrow/kratos-selfservice-ui-vue3-typescript/files/8589455/build-error.log).

```bash
error TS2345: Argument of type 'import("[repo]/node_modules/@ory/kratos-client/dist/api").UiNode' is not assignable to parameter of type 'import("[repo]/node_modules/@ory/client/dist/api").UiNode'.
  Types of property ''attributes'' are incompatible.
    Type 'import("[repo]/node_modules/@ory/kratos-client/dist/api").UiNodeAttributes' is not assignable to type 'import("[repo]/node_modules/@ory/client/dist/api").UiNodeAttributes'.
```

--- a/src/components/flows/OryFlow.vue
+++ b/src/components/flows/OryFlow.vue
@@ -25,7 +25,7 @@ import type {
   SelfServiceRecoveryFlow,
   SelfServiceSettingsFlow,
   SelfServiceVerificationFlow,
-} from '@ory/kratos-client';
+} from '@ory/client';
 import OryUiNode from './OryUiNode.vue';
 import OryUiMessage from './OryUiMessage.vue';
 import { getNodeId } from '@ory/integrations/ui';

--- a/src/components/flows/OryUiMessage.vue
+++ b/src/components/flows/OryUiMessage.vue
@@ -4,7 +4,7 @@
 </template>
 
 <script setup lang="ts">
-import type { UiText } from '@ory/kratos-client';
+import type { UiText } from '@ory/client';
 
 defineProps<{
   message: UiText;

--- a/src/components/flows/OryUiNode.vue
+++ b/src/components/flows/OryUiNode.vue
@@ -1,5 +1,5 @@
 <script setup lang="ts">
-import type { UiNode } from '@ory/kratos-client';
+import type { UiNode } from '@ory/client';
 import {
   isUiNodeAnchorAttributes,
   isUiNodeImageAttributes,

--- a/src/components/flows/OryUiNodeInput.vue
+++ b/src/components/flows/OryUiNodeInput.vue
@@ -1,5 +1,5 @@
 <script setup lang="ts">
-import type { UiNodeInputAttributes, UiNodeMeta } from '@ory/kratos-client';
+import type { UiNodeInputAttributes, UiNodeMeta } from '@ory/client';
 import TextInput from '../forms/TextInput.vue';
 import FormButton from '../forms/FormButton.vue';
 

--- a/src/plugins/ory.ts
+++ b/src/plugins/ory.ts
@@ -1,7 +1,7 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
 import type { Plugin, InjectionKey } from 'vue';
-import { Configuration, V0alpha2Api } from '@ory/kratos-client';
-import type { Session, SelfServiceLogoutUrl } from '@ory/kratos-client';
+import { Configuration, V0alpha2Api } from '@ory/client';
+import type { Session, SelfServiceLogoutUrl } from '@ory/client';
 import type { AxiosResponse } from 'axios';
 
 export const Ory = new V0alpha2Api(

--- a/src/views/LoginView.vue
+++ b/src/views/LoginView.vue
@@ -13,7 +13,7 @@
 </template>
 
 <script setup lang="ts">
-import type { SelfServiceLoginFlow } from '@ory/kratos-client';
+import type { SelfServiceLoginFlow } from '@ory/client';
 import type { AxiosError } from 'axios';
 
 import { ref } from 'vue';

--- a/src/views/RecoveryView.vue
+++ b/src/views/RecoveryView.vue
@@ -10,7 +10,7 @@
 </template>
 
 <script setup lang="ts">
-import type { SelfServiceRecoveryFlow } from '@ory/kratos-client';
+import type { SelfServiceRecoveryFlow } from '@ory/client';
 import type { AxiosError } from 'axios';
 
 import { ref } from 'vue';

--- a/src/views/RegistrationView.vue
+++ b/src/views/RegistrationView.vue
@@ -10,7 +10,7 @@
 </template>
 
 <script setup lang="ts">
-import type { SelfServiceRegistrationFlow } from '@ory/kratos-client';
+import type { SelfServiceRegistrationFlow } from '@ory/client';
 import type { AxiosError } from 'axios';
 
 import { ref } from 'vue';

--- a/src/views/SettingsView.vue
+++ b/src/views/SettingsView.vue
@@ -10,7 +10,7 @@
 </template>
 
 <script setup lang="ts">
-import type { SelfServiceSettingsFlow } from '@ory/kratos-client';
+import type { SelfServiceSettingsFlow } from '@ory/client';
 
 import { ref } from 'vue';
 import { injectStrict } from '../utils';

--- a/src/views/VerificationView.vue
+++ b/src/views/VerificationView.vue
@@ -10,7 +10,7 @@
 </template>
 
 <script setup lang="ts">
-import type { SelfServiceVerificationFlow } from '@ory/kratos-client';
+import type { SelfServiceVerificationFlow } from '@ory/client';
 
 import { ref } from 'vue';
 import { injectStrict } from '../utils';

